scripts/ci-wrapper

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+sudo ./scripts/bootstrap
+git fetch origin ${1}
+git checkout origin/${1} ./assets/index.yaml || true
+git checkout origin/${1} ./sha256sum || true
```

Comment:
good catch, I will change it.

Reply:
actually.... I am thinking about that this might not need to be changed. We don't expect this to be run locally and if it is run by github action then there should only be one remote call origin. So leave it unchanged.